types/vote_set_test.go

```diff
@@ -4,9 +4,9 @@ import (
 	"bytes"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
new line between external and internal dependencies

```suggestion
	"github.com/stretchr/testify/assert"

```